### PR TITLE
Added a donator item.

### PR DESCRIPTION
**Pull request by Useroth**
image

:cl:
add: Added a donator item
refactor: Moved some GAGS config files that I stumbled upon, while adding the above, into a more logical location
/:cl:

### CODE REVIEW

**Comment by YakumoChen** [comment]
I get that this is a donator item but it's just making an existing item into a GAGS item. Why not just make the existing heels available as GAGS?

**Comment by Draggeru** [comment]
> I get that this is a donator item but it's just making an existing item into a GAGS item. Why not just make the existing heels available as GAGS?

I'm the donator in question, my request was the heels as gags AND to add the dominant heel step sound effect

**Comment by YakumoChen** [comment]
Right. The correct move would be to replace the existing, public heels with this GAGs (the lewd heels I think?), then make the donator item a child with the sound effect

Although I thought the existing heels also had the sound effect, which would make this less of a donator request and more of a code improvement request. 

Locking a code improvement behind donator status is not the way to go about this regardless.

**Comment by Draggeru** [comment]
> Right. The correct move would be to replace the existing, public heels with this GAGs (the lewd heels I think?), then make the donator item a child with the sound effect
> 
> Although I thought the existing heels also had the sound effect, which would make this less of a donator request and more of a code improvement request.
> 
> Locking a code improvement behind donator status is not the way to go about this regardless.

I am 100% okay with this and encourage it as I do like to share!

**Comment by Useroth** [comment]
Implemented the changes; made sure it still works
image

File: modular_skyrat/modules/customization/modules/clothing/~donator/donator_clothing.dm [hunk 1]
```diff
@@ -1286,3 +1286,10 @@
 	icon = 'modular_skyrat/master_files/icons/donator/obj/clothing/uniform.dmi'
 	worn_icon = 'modular_skyrat/master_files/icons/donator/mob/clothing/uniform.dmi'
 	icon_state = "tactichill"
+
+// Donation reward for thedragmeme
+/obj/item/clothing/shoes/fancyheels/drag
```
Comment (Tastyfish) [inline modular_skyrat/modules/customization/modules/clothing/~donator/donator_clothing.dm]:
```suggestion
/obj/item/clothing/shoes/fancy_heels/drag
```

File: modular_skyrat/modules/customization/modules/clothing/~donator/donator_clothing.dm [hunk 2]
```diff
@@ -1286,3 +1286,10 @@
...
+
+/obj/item/clothing/shoes/fancyheels/drag/Initialize(mapload)
```
Comment (Tastyfish) [inline modular_skyrat/modules/customization/modules/clothing/~donator/donator_clothing.dm]:
```suggestion
/obj/item/clothing/shoes/fancy_heels/drag/Initialize(mapload)
```

**Review by Zonespace27 — APPROVED** [review]
Looks fine to me

File: modular_skyrat/modules/loadouts/loadout_items/donator/personal/donator_personal.dm [hunk 3]
```diff
@@ -541,3 +541,8 @@
 	name = "Kitsuhana Singularity Cigarettes"
 	item_path = /obj/item/storage/fancy/cigarettes/khi
 	ckeywhitelist = list("ultimarifox")
+
+/datum/loadout_item/shoes/dragheels
+	name = "Fancy Heels"
+	item_path = /obj/item/clothing/shoes/fancyheels/drag
```
Comment (Tastyfish) [inline modular_skyrat/modules/loadouts/loadout_items/donator/personal/donator_personal.dm]:
```suggestion
	item_path = /obj/item/clothing/shoes/fancy_heels/drag
```

**Review by Tastyfish — CHANGES_REQUESTED** [review]
I'm going to trust that the greyscale stuff renders correctly in-game.

However, you made it fancy_heels

**Review by Tastyfish — APPROVED** [review]

**Pull request merged**